javascript/index.js

```diff
@@ -66,3 +66,39 @@ window.onload = function onload() {
     }
   }
 };
+
+// Navigation section
+
+function navSection(navItem) {
+  const booklist = document.getElementById('bookList');
+  const addbook = document.getElementById('addBook');
+  const cont = document.getElementById('contact');
+
+  if (navItem === 'bookList') {
+    booklist.style.display = 'block';
+    addbook.style.display = 'none';
+    cont.style.display = 'none';
+  } else if (navItem === 'addBook') {
+    addbook.style.display = 'block';
+    booklist.style.display = 'none';
+    cont.style.display = 'none';
+  } else {
+    cont.style.display = 'block';
+    booklist.style.display = 'none';
+    addbook.style.display = 'none';
+  }
+}
+navSection();
+
+function dateTime() {
+  // eslint-disable-next-line no-undef
+  const luxonDate = luxon.DateTime.local().setZone('UTC+3');
+  const dateFormat = `${luxonDate.monthLong} ${luxonDate.day}th ${luxonDate.year}, ${luxonDate.hour}:${luxonDate.minute}:${luxonDate.second} `;
+  console.log(dateFormat);
+  document.getElementById('timeDiv').innerHTML = `${dateFormat}`;
+}
```

Comment:
- [ ] It would be better if the current time that is showing is the current time from the computer that is displaying the app. Currently, the time that is showing is `UTC+3`. Kindly fix this. 
- [ ] Kinsly delete the console.log() because every time the time update is showing in the console this will slow down the app and could crash the app. 

```suggestion
function dateTime() {
  // eslint-disable-next-line no-undef
  const luxonDate = luxon.DateTime.local();
  const dateFormat = `${luxonDate.monthLong} ${luxonDate.day}th ${luxonDate.year}, ${luxonDate.hour}:${luxonDate.minute}:${luxonDate.second} `;
  document.getElementById('timeDiv').innerHTML = `${dateFormat}`;
}
```

Reply:
Hello @BrianSammit

The timezone show is my local time zone (+3), would want me to change it program time zone to (-6) instead? please let me know how to proceed.